.github/workflows/test_and_build_image.yaml

```diff
@@ -31,12 +31,10 @@ jobs:
         with:
           go-version-file: 'go.mod'
           cache: true
-      - name: Create env variables
-        run: ./.github/scripts/env_variables.sh
       - name: Install yq
         run: |
           sudo rm /bin/yq
-          wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64.tar.gz
+          wget https://github.com/mikefarah/yq/releases/download/v4.11.2/yq_linux_amd64.tar.gz
```

Comment:
Still installing yq? An ancient one even?

Reply:
yes, otherwise our unit tests [will fail](https://github.com/k8ssandra/k8ssandra-operator/issues/820).
I'm not planning on fixing this as part of this PR.